content/includes/homepage-warning.html

```diff
@@ -1,9 +1,9 @@
 <div id="top-of-page-alert-container">
   <div id="top-of-page-alert" class="usa-alert usa-alert-warning">
       <div class="usa-alert-body">
-          <h4 class="usa-alert-heading">Some Vets.gov tools may not be working</h4>
-          <p class="usa-alert-text">
-              We’re sorry. We’re working on our data system right now. Some Vets.gov tools—including Veteran ID cards, claims and appeals, and Post-9/11 GI Bill benefits status—may not be working during this time. Please check back soon.
+          <h4 class="usa-alert-heading">We'll Continue to Provide Most Services During the Shutdown</h4>
```

Comment:
Do we normally title case these kinds of headlines?

Reply:
Yes, that was from @bethpotts plain language and Vets.gov style/tone guidance